tools/rimage/config/ptl.toml.h

```diff
@@ -0,0 +1,124 @@
+#include "platform-ptl.toml"
+
+[[module.entry]]
+name = "BRNGUP"
+uuid = "2B79E4F3-4675-F649-89DF-3BC194A91AEB"
+affinity_mask = "0x1"
+instance_count = "1"
+domain_types = "0"
+load_type = "0"
+module_type = "0"
+auto_start = "0"
+
+index = __COUNTER__
+
+[[module.entry]]
+name = "BASEFW"
+uuid = "0E398C32-5ADE-BA4B-93B1-C50432280EE4"
+affinity_mask = "3"
+instance_count = "1"
+domain_types = "0"
+load_type = "0"
+module_type = "0"
+auto_start = "0"
+
+index = __COUNTER__
+
+#ifdef CONFIG_COMP_MIXIN_MIXOUT
+#include <audio/mixin_mixout/mixin_mixout.toml>
+#endif
+
+#ifdef CONFIG_COMP_COPIER
+#include <audio/copier/copier.toml>
+#endif
```

Comment:
So... manifest construction and TOML files aren't really my area, but I have a hard time believing this giant tree of "include all the component types in the build" is best expressed in a cut/paste copy of files in every different subplatform.  I mean, pretty clearly there's no new PTL code here.

Can't you put this somewhere in a single file, and include it from everyone that needs it?

Reply:
> Can't you put this somewhere in a single file, and include it from everyone that needs it?

Indeed, I think that's why we added .toml preprocessing in `west sign`.